raiden_installer/web.py

```diff
@@ -620,7 +665,8 @@ def post(self, configuration_file_name):
         [
             url(r"/", IndexHandler, name="index"),
             url(r"/configurations", ConfigurationListHandler, name="configuration-list"),
-            url(r"/setup/(mainnet|goerli)", SetupHandler, name="setup"),
+            url(r"/setup/(mainnet|goerli)/(.*)", SetupHandler, name="setup"),
+            url(r"/create_wallet", WalletCreationHandler, name="create_wallet"),
```

Comment:
where is the endpoint `create_wallet` called?

Reply:
[here](https://github.com/raiden-network/raiden-wizard/pull/148/files#diff-0ee0c1487407917328c8cac52295d9e7R93)

Reply:
But I mean, who is calling create wallet? Other than the redirect from the setupHandler I cannot see it. So Why not calling it directly from the frontend?